### PR TITLE
[Console] Use UTF-8 bullet for listing

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no     
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #...   
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Inspired by the new table style, this updates the listing style.

before

image

after

image

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Be careful. That's one of the multiple characters that don't work on Windows. See https://github.com/sindresorhus/figures

File: src/Symfony/Bundle/FrameworkBundle/Tests/Console/Descriptor/AbstractDescriptorTest.php [hunk 1]
```diff
@@ -241,11 +241,22 @@ private function getEventDispatcherDescriptionTestData(array $objects)
...
             }
         }
 
         return $data;
     }
+
+    private function getFixtureFilename($name)
+    {
+        $format = $this->getFormat();
+        $file = ($baseDir = __DIR__.'/../../Fixtures/Descriptor').'/'.$name.'.'.$format;
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Console/Descriptor/AbstractDescriptorTest.php]: can you move the `$baseDir` assignement to its own statement ? Nested assignments are not that readable.

**Comment by ro0NL** [comment]
Now ready :)

File: src/Symfony/Bundle/FrameworkBundle/Tests/Console/Descriptor/AbstractDescriptorTest.php (hunk 1 above)
Comment (ro0NL) [reply src/Symfony/Bundle/FrameworkBundle/Tests/Console/Descriptor/AbstractDescriptorTest.php]: Done.

**Comment by ogizanagi** [comment]
Duplicating expected outputs for windows may be unwanted. Can't we use matchesFormat or str_replace bullets instead (and create a dedicated test case for this behavior)?

**Comment by ro0NL** [comment]
the files are globbed.. i found this the easiest, To maintain and be future proof. Not aiming for in-test formatting hassle / excluding these fixtures into separate tests.

**Review by fabpot — APPROVED** [review]

**Review by ogizanagi — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ro0NL.

**Comment by fabpot** [comment]
reverted, see discussion on #26718

**Pull request merged**